.github/workflows/ci.yml

```
@@ -8,6 +8,15 @@ env:
 jobs:
   build:
     runs-on: windows-2022
+    needs: [format-check, format-check-cmake-files]
+    strategy:
```

```suggestion
    runs-on: windows-2022
    strategy:
```

Trying to sneak in #720? Hehe :>